importer/api/__init__.py

```diff
@@ -148,7 +148,7 @@ def import_children(course, element, parent, parent_dpath):
         None
     """
     # pylint: disable=too-many-locals
-    title = element.attrib.get("display_name", "MISSING")
+    title = element.attrib.get("display_name", "...")
```

Comment:
no, I mean the ellipsis you put is a sequence of 3 dots and not the unicode ellipsis character U+2026